content/courses/tokens-and-nfts/nfts-with-metaplex.md

```diff
@@ -426,18 +492,21 @@ The output should look like this:
 
 ```
 % npx esrun create-metaplex-nft-collection.ts
-Loaded user: 4DRi8LxWhAWf9LwUuyb3Rz4Y4USEKzjdvQBWTyrkJtwg
-image uri: https://arweave.net/_3vbKzFO7zcfMkFaVdhorntkPHDqz4YNTyf_9lrGw1c
-Collection offchain metadata URI: https://arweave.net/Jw29lMkHp-PIxddEuLFkavdOTP1bJ9MeXTu-QqY3qTM
-Collection NFT: https://explorer.solana.com/address/8dP75EWHnYAc6pgeiToXtE66qEaHx1PcoetZpmnAQeAp?cluster=devnet
-Collection NFT address is 8dP75EWHnYAc6pgeiToXtE66qEaHx1PcoetZpmnAQeAp
+
+Loaded user: 4kg8oh3jdNtn7j2wcS7TrUua31AgbLzDVkBZgTAe44aF
+image uri: https://arweave.net/XWpt7HDOFC0wJQcQWgP9n_cxHS0qQik9-27CAAaGP6E
+Collection offchain metadata URI: https://arweave.net/atIf58t3FHa3heoOtNqPkVvEGC_9WzAduY0GQE-LnFI
+Collection NFT:  https://explorer.solana.com/address/D2zi1QQmtZR5fk7wpA1Fmf6hTY2xy8xVMyNgfq6LsKy1?cluster=devnet
+Collection NFT address is: D2zi1QQmtZR5fk7wpA1Fmf6hTY2xy8xVMyNgfq6LsKy1
 ✅ Finished successfully!
 ```
 
-Congratulations! You've created a Metaplex Verified Collection. Check this out
-on Solana Explorer using the URL above. If you have any trouble, try and fix it
-yourself, but if you need to you can also check out the
-[solution code](https://github.com/solana-developers/professional-education/blob/main/labs/create-metaplex-nft-collection.ts).
+Congratulations! You've created a Metaplex Collection. Check this out on Solana
+Explorer using the URL above.
+
+If you have any trouble, try and fix it yourself, but if you need to you can
+also check out the
+[solution code](https://github.com/solana-developers/professional-education/blob/main/labs/metaplex-umi/create-collection.ts).
```

Comment:
Add an image here of what Explorer should look like. See CONTRIBUTING for image specifications.